src/game/board.rs

```diff
@@ -55,14 +55,14 @@ impl BoardHandler {
     pub fn new(board_width: u8, board_height: u8, num_players: u8, mode: GameMode) -> Self {
         // determine some rules based on gamemode
         let (board_height_buffer, spawn_row) = match mode {
-            GameMode::None => panic!(BH_MODE_NONE),
+            GameMode::None => panic!("{}", BH_MODE_NONE),
```

Comment:
If I remember when I read this code last, all of these panics are probably better defined as `unreachable!()`